feat(panel): add never-consent feedback

smalluban opened the pull request.

philipp-classen commented on src/content_scripts/autoconsent.js:
```diff
@@ -11,11 +11,38 @@
 
 import AutoConsent from '@duckduckgo/autoconsent';
 
+const FLAG_STORAGE_KEY = 'ghostery:autoconsent';
```
One problem that I see is that it can be used for adblocker detection (by anti-adblockers) and for fingerprinting (trackers).

I'm not aware of a straightforward technique to transfer information in a safe and non-detectable way. So, I cannot offer an constructive advice. Message passing might be the best choice.

smalluban replied:
That's a good point, I didn't take into account. Ill search if there is another way to store information in sync with browsing history of the user.

smalluban commented:
I moved the storage to the internal one using the `Resources` model. I realized that keeping the flag, even though the CMP can be shown again, is not completely wrong. If a user clears browsing history (or cookies expire), the never-consent will do it again, so the information is still valid. Most importantly, the information is no longer stored in the page world.

I also added a condition to hide the feedback if the extension is paused or never-consent is disabled.
